### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by dsyme** [fsprojects/FSharp.Compiler.CodeDom#16]
@drvink The build is broken, both on Windows and on Travis/Linux, I think in both cases because of the installer

```
C:\GitHub\fsprojects\FSharp.Compiler.CodeDom\Installer\ModifyMachineConf
ig\ModifyMachineConfig.csproj(71,3): error MSB4019: The imported project
 "C:\Program Files (x86)\MSBuild\Microsoft\WiX\v3.x\Wix.CA.targets" was
not found. Confirm that the path in the <Import> declaration is correct,
 and that the file exists on disk.
```

**Comment by dsyme** [comment]
(I wonder if the installer should go in a separate repo)

**Comment by drvink** [comment]
I definitely would prefer *not* to split it out into another repo, since the maintenance cost would increase dramatically. Disabling it by default is probably the easiest, though it seems kind of clumsy to make users have to edit the solution file in order to build it...any ideas?

**Comment by dsyme** [comment]
Put it in a separate solution then, which is not built by default?

**Comment by drvink** [comment]
That's what I was thinking too, just wanted your opinion. I guess `build.cmd` should take an optional argument which if present will pass `-d:BUILD_INSTALLER` when running `build.fsx`, like how `build.sh` does with `-d:MONO`.

**Comment by drvink** [comment]
Should be fixed now. I tested every combination. Sorry for not realizing it was broken!

**Comment by dsyme** [comment]
Travis build still looks like it's failing? : https://travis-ci.org/fsprojects/FSharp.Compiler.CodeDom/builds/120804583

Cheers!
don

**Comment by drvink** [comment]
I don't know anything about Travis, but it looks like at one point the builds were being done on OS X, but now they're being done on Linux (?!) and that's why the build is failing, since it doesn't even get as far as being able to install Mono.

**Comment by drvink** [comment]
Fixed by #17. Thanks @sergey-tihon !
